### ISSUE TITLE
Breaks if no kernings?

### ISSUE THREAD

**Issue by CodeLenny** [mattdesl/parse-bmfont-xml#1]
I tried using parse-bmfont-xml with a font created with [Littera](http://kvazars.com/littera/), and got the following error:

```
     TypeError: Cannot read property '0' of undefined
    at node_modules/parse-bmfont-xml/lib/index.js:39:36
```

I traced that back to a part of the code that dealt with kernings, and realized that my font didn't include a kernings section.

I'm not familiar with the font format, Is having such a section a requirement for a font?  Even if it is, would it be wise to check if `kernings` exists?

I got around it by adding

```xml
<kernings count="0"></kernings>
```

to my font file, but I had to do a bit of digging to find out what was causing the error.

**Comment by mattdesl** [comment]
Thanks for testing ! Not all fonts have kernings so it would be best to do a check for that. I will accept a PR to fix. 😄

**Comment by CodeLenny** [comment]
Cool!  I'll see if I can get something written in the near future.  I've been PRing a ton today.

**Comment by mattdesl** [comment]
Should be fixed now in 1.1.4
